<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>731</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget2{
	background-color: rgba(60,60,60,255);
	border-radius: 10px;
}</string>
   </property>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Toevoegen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Verwijder</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="LPButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>LP's</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bookButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Books</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="gameButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>92</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Games</string>
    </property>
   </widget>
   <widget class="QListView" name="search_items">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>341</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Name of Item</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="iets1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>u mama of Item</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="iets2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Nog iets of &gt;item</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="iets3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>380</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>vagina oof item</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="bobs">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>bobs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
